pkgs/development/libraries/libxml2/default.nix

```diff
@@ -34,16 +35,19 @@ in
 let
 libxml = stdenv.mkDerivation rec {
   pname = "libxml2";
-  version = "2.12.3";
+  version = "2.12-unstable-2023-12-14";
```

Comment:
Should not this be `2.12.3-unstable-2023-12-14` when it includes `2.12.3`?

Reply:
Yes.  Fixed.